Lib/glyphsLib/builder.py

```diff
@@ -649,10 +654,11 @@ def load_glyph(glyph, layer, glyph_data):
     if export is not None:
         glyph.lib[glyphlib_prefix + 'Export'] = export
     production_name = glyph_data.get('production')
+    extra_glyph_info = None
     if production_name is None:
-        glyphinfo = glyphsLib.glyphdata.get_glyph(glyph.name)
-        if glyphinfo:
-            production_name = glyphinfo.production_name
+        extra_glyph_info = glyphdata_generated.DEFAULT_GLYPH_DICT.get(glyph.name)
```

Comment:
why do we have a `get_glyph` function if we don't use it here to look up the production names from `glyphdata_generated`?